manifests/m/Microsoft/SQLServerManagementStudio/19.0/Microsoft.SQLServerManagementStudio.installer.yaml

```diff
@@ -0,0 +1,138 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+PackageIdentifier: Microsoft.SQLServerManagementStudio
+PackageVersion: '19.0'
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+FileExtensions:
+- SQL
+ReleaseDate: 2023-01-26
+Installers:
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerType: burn
+  Scope: machine
+  InstallerUrl: https://go.microsoft.com/fwlink/?linkid=2223041&clcid=0x407
+  InstallerSha256: C067BD6D092F4971C64CC17F9ABF0FDC8A83DEFBDE987DEE17F12ADD5FE7A36E
+  AppsAndFeaturesEntries:
+  - DisplayName: Microsoft SQL Server Management Studio - 19.0
+    DisplayVersion: 
+    ProductCode: '19.0.20196.0'
```

Comment:
```suggestion
```
This value is incorrectly put under `ProductCode` and should be under `DisplayVersion`. It's the same case for every installer node in the manifest.

Reply:
Same as the other one: they came from the same script.